src/Cli/func/Actions/LocalActions/GetExtensionBundlePath.cs

```diff
@@ -25,7 +25,11 @@ public override async Task RunAsync()
             {
                 try
                 {
-                    string bundlePath = await extensionBundleManager.GetExtensionBundlePath();
+                    var options = ExtensionBundleHelper.GetExtensionBundleOptions();
```

Comment:
I thought `ExtensionBundleManager.GetExtensionBundlePath` was meant to be the single source of truth for getting the extension bundle path. Is there a reason we are not using it here? Is that method not returning the expected output in some scenarios? Trying to understand the switch to the new logic here.

Reply:
The problem is that `GetExtensionBundlePath` will also install templates if they don't exist. Given this action is just to return the path itself it doesn't make sense for it to also download the bundles. I believe this is also one of the contributors for the flakeyness happening in the e2e tests. 

Second, `GetExtensionBundlePath` uses the download path that it gets from core tools so this is just shortcuriting all the download bundle stuff that we don't need here.